src/main/java/duke/tasktypes/Deadlines.java

```diff
@@ -0,0 +1,50 @@
+package duke.tasktypes;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Deadline class.
+ */
+public class Deadlines extends Task {
```

Comment:
I didn't like the fact that Task is a singular noun but the child class extending from it is plural. Do you have any rationale for naming these classes in their plural form?